tests/test_graphband.py

```
@@ -3,342 +1,449 @@
+        w1 = session.get(WorkerEntry, proc.pid)
+        assert w1 is not None
+        assert w1.status == WorkerStatus.KILLED
+        assert len(w1.running_tasks) == 0
+        w2 = session.get(WorkerEntry, os.getpid())
+        assert w2 is not None
+        assert w2.status == WorkerStatus.OFFLINE
+        assert len(w2.running_tasks) == 0
+
+        tasks = session.query(TaskEntry).all()
+        assert len(tasks) == 10
+        tasks[0].current_status.status == TaskStatusEnum.KILLED
```

Ruff (0.12.2)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)